cmd/bbolt/main.go

```diff
@@ -1103,17 +1103,33 @@ func (cmd *benchCommand) Run(args ...string) error {
 	db.NoSync = options.NoSync
 	defer db.Close()
 
+	r := rand.New(rand.NewSource(time.Now().UnixNano()))
+
 	// Write to the database.
 	var writeResults BenchResults
+	var keys []nestedKey
+
+	if options.ReadMode == "rnd" {
+		keys = make([]nestedKey, 0, options.Iterations)
+	} else {
+		keys = nil
+	}
```

Comment:
Note: The value of an uninitialized slice is nil.

```suggestion
	} 
```